src/components/Guilds/ActionsModal/ContractsList.tsx

```diff
@@ -47,7 +47,7 @@ const ContractsList: React.FC<ContractsListProps> = ({
             Transfer & Mint
           </ButtonLabel>
         </ActionsButton>
-        {isRepGuild ? (
+        {isSnapshotRepGuild ? (

```

Comment:
The previous isRepGuild is actually correct. We don't care about snapshot feature here